### PR TITLE
Compute polygon centroid instead of average of positions

### PR DESCRIPTION
**Pull request by hpinkos**
For `PolygonGeometryUpdater._computeCenter` we were doing a fast approximation of the center by averaging the positions, but it was requested that we compute the true polygon centroid instead.

I also added `EllipsoidTangentPlane.projectPointOntoEllipsoid`.  We already had `projectPointsOntoEllipsoid` but I wanted to only project a single point.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @hpinkos!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/polygon-centroid/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/DataSources/PolygonGeometryUpdater.js [hunk 1]
```diff
@@ -216,17 +222,30 @@ define([
...
+
+            var sum = Cartesian2.add(p1, p2, cart2Scratch);
+            sum = Cartesian2.multiplyByScalar(sum, f, sum);
+            centroid2D = Cartesian2.add(centroid2D, sum, centroid2D);
+
+            area += f;
         }
-        return centroid;
+
+        var a = 1 / (area * 6);
```
Comment (bagnell) [inline Source/DataSources/PolygonGeometryUpdater.js]: Shouldn't this be `var a = 1.0 / (area * 3.0);`?

Comment (hpinkos) [reply Source/DataSources/PolygonGeometryUpdater.js]: Yes, you're right. Thanks

**Comment by hpinkos** [comment]
@bagnell ready

File: Source/DataSources/PolygonGeometryUpdater.js [hunk 2]
```diff
@@ -216,17 +222,30 @@ define([
...
+
+            var sum = Cartesian2.add(p1, p2, cart2Scratch);
+            sum = Cartesian2.multiplyByScalar(sum, f, sum);
+            centroid2D = Cartesian2.add(centroid2D, sum, centroid2D);
+
+            area += f;
         }
-        return centroid;
+
+        var a = 1 / (area * 3);
```
Comment (bagnell) [inline Source/DataSources/PolygonGeometryUpdater.js]: Use `1.0` and `3.0` for floating point numbers. Same with the new tests.

**Comment by hpinkos** [comment]
@bagnell updated

**Pull request merged**